src/couch_views/src/couch_views_indexer.erl

```diff
@@ -29,6 +29,8 @@
 % TODO:
 %  * Handle timeouts of transaction and other errors
 
+-define(KEY_SIZE_LIMIT, 8000).
```

Comment:
Maybe rename it to `DEFAULT_KEY_SIZE` and `DEFAULT_VALUE_SIZE_LIMIT`

Reply:
Done.